server.go

```diff
@@ -1944,6 +1953,33 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 	return nil
 }
 
+// SetSendCompressor sets the compressor that will be used when sending
```

Comment:
Nit: simplify the first sentence as follows:

// SetSendCompressor sets a compressor for outbound messages.

Also, please mark this API experimental.